### PR TITLE
add list with titles

### PR DESCRIPTION
**Pull request by jauniuse**

### CODE REVIEW

File: src/Molecules/ListWithTitles/ListWithTitles.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { Flexbox, ListItem } from '../..';
+import ListWithTitles from '.';
```
Comment (shmidt-i) [inline src/Molecules/ListWithTitles/ListWithTitles.stories.tsx]: Import this from '../..' as well

File: src/Molecules/ListWithTitles/ListWithTitles.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+const ItemContainer: React.FunctionComponent<{ children: React.ReactNode }> = ({ children }) => (
```
Comment (shmidt-i) [inline src/Molecules/ListWithTitles/ListWithTitles.stories.tsx]: You can skip children here, they are already part of React.FunctionComponent

File: src/Molecules/ListWithTitles/ListWithTitles.stories.tsx (hunk 1 above)
Comment (jauniuse) [reply src/Molecules/ListWithTitles/ListWithTitles.stories.tsx]: updated

File: src/Molecules/ListWithTitles/ListWithTitles.stories.tsx (hunk 2 above)
Comment (jauniuse) [reply src/Molecules/ListWithTitles/ListWithTitles.stories.tsx]: updated

**Pull request merged**